web-components/.storybook/preview.js

```diff
@@ -0,0 +1,29 @@
+import UswdsTheme from "./UswdsTheme";
+
+// Global component styles.
+import "../src/index.css";
+// Theme overrides.
+import "./index.css";
+
+
+/** @type { import('@storybook/web-components').Preview } */
+const preview = {
+  parameters: {
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/i,
+      },
+    },
+    docs: {
+      toc: true, // Autogenerate table of contents.
+      theme: UswdsTheme,
+      canvas: {
+        sourceState: 'shown'
```

Comment:
> [!NOTE]  
> **Not a blocker**, but indentation can be off in some previews. There's a workaround in this issue [DocsPage code preview indentation is wrong · Issue #8078 · storybookjs/storybook](https://github.com/storybookjs/storybook/issues/8078).

Reply:
FYI, I got this working on the MD WDS project I'm working on!

Here are my notes on the workaround:
https://github.com/storybookjs/storybook/issues/8078#issuecomment-2325332120

Reply:
@caseywatts thanks for sharing. Created an issue to give this a try 😄